testing/catalog.go

```diff
@@ -1172,3 +1173,19 @@ func (c *Catalog) NatsSecret(deployName string) corev1.Secret {
 		},
 	}
 }
+
+// WaitingPod is a pod which has the wait-for annotation for use in tests
+func (c *Catalog) WaitingPod(name, service string) corev1.Pod {
+	return c.AnnotatedPod(name, map[string]string{
+		waitservice.WaitKey: service,
+	})
+}
+
+// EchoContainer returns a container which just "echo" for use in tests
+func (c *Catalog) EchoContainer(name string) corev1.Container {
+	return corev1.Container{
+		Name:    name,
+		Image:   "busybox",
```

Comment:
```suggestion
		Image:   "opensuse/leap",
```

Reply:
No real catch to use leap here.. it's for integration tests, the smaller the better